src/components/Tooltip.vue

```diff
@@ -1,31 +1,31 @@
 <template>
     <span class="tooltip"
-        :class="{ active: tooltipActive }"
+        :class="[tooltipPosition, {
+            shown: isShown,
+            'transition-position': transitionPosition,
+            'inverse-theme': theme === constructor.Themes.INVERSE,
+        }]"
         @mouseenter="mouseOver(true)"
         @mouseleave="mouseOver(false)"
     >
         <a href="javascript:void(0);"
-            ref="tooltipIcon"
-            @focus.stop="toggleTooltip"
-            @blur.stop="tooltipToggled ? toggleTooltip() : ''"
-            :class="{
-                top: tooltipPosition === 'top',
-                bottom: tooltipPosition === 'bottom',
-            }">
-            <slot name="icon">
+            ref="tooltipTrigger"
+            @focus.stop="show"
+            @blur.stop="hide"
+            :tabindex="disabled ? -1 : false"
+            class="trigger">
+            <slot name="trigger">
```

Comment:
I think the previous names are not well chosen and therefore changed them to more descriptive ones.
The existing Tooltips need to be changed anyways, because also the default behavior for `autoWidth` and the `preferredPosition` changed compared to the previous implementation.
In `vue-components` there has been no usage of `Tooltip` so far. In other projects, I'm only aware of the one used in `CreateCashlink`.
I don't like the idea of choosing defaults and api names based only on a single Tooltip occurence, while other defaults and names make more sense looking at other tooltips in the designs which are just not implemented yet.
Therefore I'd keep the new names and defaults. Adding an additional slot just unnecessarily complicates the component.
I now added exceptions on deprecated usage of the component, but do not want to invest further time to make it backwards compatible.

Reply:
I always thought the timer uses Tooltip, but it has its own tooltip.

The tooltip is used in the donation button app and in a few places in the new wallet. 
Likely more places.

Reply:
I looked into backwards compatibility again and it was easier than thought. So `container` and `trigger` are now fully backwards compatible.